app/services/axiosDeleteToDo.js

```diff
@@ -0,0 +1,14 @@
+// services a fianco alla cartella containers
+import axios from 'axios';
```

Comment:
puoi rimuovere axios dal nome del file e riumovere anche axios da qui!

Puoi creare un file chiamato networkProxy.js ad esempio dove dentro importi axios, crei un'instanza, la configuri e la esporti.

Poi dai servizi fai `import proxy from utils/networkProxy.js` e usi proxy cosi come lo stai usando ora.

Reply:
Anche perche' facendo cosi come fai ora stai setti il baseurl in ogni servizio, invece l'idea del proxy è quella di avere un delegato per il networking preconfigurato.